rasa/nlu/tokenizers/whitespace_tokenizer.py

```diff
@@ -14,17 +14,13 @@ class WhitespaceTokenizer(Tokenizer):
         "intent_tokenization_flag": False,
         # Symbol on which intent should be split
         "intent_split_symbol": "_",
-        # Text will be tokenized with case sensitive as default
-        "case_sensitive": True,
     }
 
     def __init__(self, component_config: Dict[Text, Any] = None) -> None:
         """Construct a new tokenizer using the WhitespaceTokenizer framework."""
 
         super().__init__(component_config)
 
-        self.case_sensitive = self.component_config["case_sensitive"]
```

Comment:
should we add a deprecation warning if this option is present in the config to move it to featurizers?

Reply:
Not sure if it is needed as it goes into Rasa 2.0, but can add a quick warning, that this option is not present anymore.